internal/manifest/manifest.go

```diff
@@ -2,12 +2,20 @@ package manifest
 
 import (
 	"encoding/xml"
+	"errors"
 	"fmt"
 	"io"
 	"path/filepath"
 	"slices"
+
+	"github.com/artefactual-sdps/preprocessing-sfa/internal/identifiers"
 )
 
+type File struct {
+	ID       string
+	Checksum Checksum
+}
```

Comment:
I think it's better to keep the `identifiers` package separate.  The `identifiers` package is intended to represent the structure of an `identifiers.json` file, which is only tangentially related to encoding or decoding a manifest file (e.g. `metadata.xml`).

Based on this separation I probably shouldn't have made `manifest.Identifiers()` a method in the manifest package. I think I'll move the code from `manifest.Identifiers()` to the identifiers package as a new method that takes a manifest as a parameter.

Reply:
Not a big deal either, you could see it both ways. I see them as identifiers that come from the manifest, that also have a JSON representation. Since the activity lives in another package, it would make sense to put it inside `manifest` to me.